be/src/exprs/vectorized/json_functions.h

```diff
@@ -121,6 +121,20 @@ class JsonFunctions {
      */
     DEFINE_VECTORIZED_FN(json_string);
 
+    /**
+     * @param: [json_column]
+     * @paramType: [JsonColumn]
+     * @return: Int32Column
+     */
+    DEFINE_VECTORIZED_FN(json_int);
```

Comment:
Does our user need these functions?

Reply:
`json_int`, `json_double` and `string_double` are all used internally, so I make them as private now.